packages/neuron-ui/src/components/GeneralSetting/index.tsx

```diff
@@ -135,7 +135,9 @@ const GeneralSetting = ({ updater }: GeneralSettingProps) => {
   const [showLangDialog, setShowLangDialog] = useState(false)
   const [searchParams] = useSearchParams()
   const [errorMsg, setErrorMsg] = useState('')
-  const [dialogType, setDialogType] = useState<'' | 'checking' | 'updating' | 'updated'>('')
+  const [dialogType, setDialogType] = useState<'' | 'checking' | 'checked' | 'updating' | 'updated'>('')
```

Comment:
The new type `checked` can be avoided by `const isChecked = !!updater.version`

Reply:
> The new type `checked` can be avoided by `const isChecked = !!updater.version`

In the logic of the function `handleUpdate`, there needs to a key to open our installation dialog, and on the other hand, we don't want the installation dialog to open as soon as we enter the page.